server/routes/index.js

```diff
@@ -5,6 +5,7 @@ import property from './property.routes';
 import enquiry from './enquiry.routes';
 import mailer from './mailer.routes';
 import visitation from './visitation.routes';
+import payment from './paymentPlan.routes';
```

Comment:
Lets be explicit with paymentPlan as we don't want a mix up with the payment and transaction model.

```suggestion
import paymentPlan from './paymentPlan.routes';
```